contrib/brl/bpro/core/vpgl_pro/processes/vpgl_get_bounding_box_process.cxx

```diff
@@ -88,10 +88,10 @@ bool vpgl_get_bounding_box_process(bprb_func_process& pro)
 
     //intersect each ray with z plane
     vgl_point_3d<double> ulp, urp, blp, brp;
-    bool good =    vgl_intersection(ul, zp, ulp);
-    good = good && vgl_intersection(ur, zp, urp);
-    good = good && vgl_intersection(bl, zp, blp);
-    good = good && vgl_intersection(br, zp, brp);
+    vgl_intersection(ul, zp, ulp);
+    vgl_intersection(ur, zp, urp);
+    vgl_intersection(bl, zp, blp);
+    vgl_intersection(br, zp, brp);
 
```

Comment:
Wouldn't it be better to keep the return values from vgl_intersection, but add code to actually check it and report an error if it fails?